README.md

```diff
@@ -57,6 +57,36 @@ Warning: LFS client uses HTTP basic authentication, so using **HTTPS** is a must
 ./lfs_server.sh -verbose -pam login -s IP_ADDRESS -p PORT -cert domain.crt -key domain.key
 ```
 
+## Example
+
+```
+# Start a LFS server
+./lfs_server.sh
+
+# Clone a repo
+git clone ....
+cd repo
+
+# Add a normal file
+touch test.txt
+git add test.txt
+git commit -m "normal file"
+
+# Add a lfs file 
+git lfs install
+mkfile -n 10m test.bin
```

Comment:
mkfile is not available on linux

dd is more universal way to create a file

dd if=/dev/zero of=test.bin count=10240 bs=1024